src/double.js

```diff
@@ -0,0 +1,57 @@
+const ms = require('ms');
+
+export default class DoubleDecker {
+
+  constructor (one, two) {
+    this.one = one;
+    this.two = two;
+  }
+
+  get (k, cb) {
+    return this.one.get(k, (err, res) => {
+      if (err) return cb(err);
+      if (!res) return cb(null, null);
+      if (res < Date.now()) {
+
+        // SIDEEFFECT! kickoff del -- note this is an
+        // asyhronous function, but be purposely ignore
+        // it's return and just hope it goes through...
+        this.del(this.one, this.two, k);
+        return cb(null, null);
+      }
+      return this.two.get(k, cb);
+    });
+  }
+
+  set (k, v, ttl, cb) {
```

Comment:
This is just about own preference: I prefer to use longer and more descriptive variable names, its easier to read and understand whats going on:) In this case its pretty obvious anyways though!
I.e. `set (key, value, ttl, cb)`